mlir-sycl/lib/Dialect/IR/SYCLOpsTypes.cpp

```diff
@@ -415,6 +415,73 @@ llvm::ArrayRef<mlir::Type> mlir::sycl::AccessorImplDeviceType::getBody() const {
   return getImpl()->Body;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// AccessorSubscriptType Operations
+////////////////////////////////////////////////////////////////////////////////
+
+mlir::sycl::AccessorSubscriptType
+mlir::sycl::AccessorSubscriptType::get(MLIRContext *Context,
+                                       unsigned int CurrentDimension,
+                                       llvm::SmallVector<mlir::Type, 4> Body) {
+  return Base::get(Context, CurrentDimension, Body);
+}
+
+mlir::Type
+mlir::sycl::AccessorSubscriptType::parseType(mlir::DialectAsmParser &Parser) {
+  if (mlir::failed(Parser.parseLess())) {
+    return nullptr;
+  }
```

Comment:
[nit]: Why are we using {} here?  LLVM coding conventions states that 'ifs' with single stmt bodies should not have {}. 

P.S>: I realize the other similar implementations are braced, nevertheless I'd like if we can change them all to be conforming with LLVM coding conventions.

Reply:
Great, then I will keep it unchanged for now in this PR.